.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,61 @@
+name: Bug Report
+description: Report a bug.
+labels: ["bug"]
+body:
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: Provide a clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Pprovide detailed steps to replicate the bug.
+      placeholder: |
+        1. In this environment...
```

Comment:
Four instances of "1.", assuming this should be 1,2,3,4?

Reply:
Ah was using markdown conventions seen [here](https://www.markdownguide.org/basic-syntax/#ordered-lists), but it doesn't appear to work with GitHub issue templates. Will change!